src/backend/TrafficCourts/Staff.Service/Mappers/Mapper.cs

```diff
@@ -22,6 +22,9 @@ public static DisputeApproved ToDisputeApproved(Dispute dispute)
             {
                 Count = violationTicketCount.Count,
                 Section = violationTicketCount.Section,
+                Subsection = violationTicketCount.Subsection,
+                Paragraph = violationTicketCount.Paragraph,
+                Act = violationTicketCount.Act,

```

Comment:
I didn't realise Section was split out to subsection and paragraph in the model.  I'm not sure it should be.

Here is a listing of the MVA acts we currently have:
https://github.com/bcgov/jag-traffic-courts-online/blob/main/src/backend/oracle-data-api/src/main/resources/data/statutes.csv

This is is what is used to OCR a handwritten ticket (to validate the Section text with the data in the Section column in the csv).  Here are some examples:
```
24.1(1)(b.1)(viii)
24(4) Section (3)(a)
26(1)(b.1)(x)
170(b)(iii)
```
You can see, some of these Section values don't map well to Section, Subsection, and Paragraph used by ARC.

Reply:
Yes, I know it's represented as you mentioned in the example on the citizen side and OCR reads it as single section string. However, we have to split it out to section, subsection, paragraph and act fields since ARC file has separate fixed length fields for each so I thought they should be split out at the staff portal level. That's why I added both a "fullSection" field and each field representing other subsections. I think it can be represented on staff portal front-end as separated fields like section, subsection, paragraph and act when it's retrieved from the api from "fullSection" field. So, if staff can verify whether it's accurate and approves the dispute, then it could be mapped to the separated fields on oracle-api and published to the message queue for ARC.

Reply:
Yeah, I think this needs discussion.  I realize that ARC needs them split out, but I think this logic should be in the ARC message consumer, not in the front-end.  Also, the MVA full Section data is often 4 groups, not three so there is a disparity between what structure ARC is expecting and the structure that is currently in JUSTIN.

Reply:
In my opinion, it would be safer and more standardized if we can verify the split out data through the staff portal but yes, you're right. I agree that this needs a discussion.

Reply:
So it sounds like the decision is to have multiple columns in Oracle.  I think we need at least 4 columns for now: section, subsection, paragraph, and clause - maybe a 5th for subclauses later.
From my understanding, for now, ARC will just use the first 3 columns.